### PR TITLE
Feature/#3245 implement certificates page

### PR DESCRIPTION
**Pull request by s1auka**
Added table with certificates on UBS-admin "Certificates" tab

### CODE REVIEW

File: src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.scss [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+.research {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 426px;
+  height: 36px;
+  border: 1px solid var(--ubs-primary-light-grey);
+  border-radius: 4px;
+  margin-top: 20px;
+  margin-bottom: 32px;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.scss:180]: Combine this into one:
margin: 20px 0 32px 0

File: src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.scss [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+  }
+}
+
+.dropdown-menu {
+  height: 480px;
+  min-width: 356px;
+  margin-right: 5px;
+  overflow: scroll;
+  padding: 24px;
+  margin-top: 170px;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.scss:249]: Combine this into one:
margin: 170px 5px 0 0

File: src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.scss [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+.dropdown-menu input {
+  width: 16px;
+  z-index: 9;
+  border-radius: 3px;
+  border: 1px solid var(--ubs-secondary-grey);
+}
+
+.mat-header-cell .mat-icon {
+  padding-top: 5px;
+  padding-left: 10px;
```
Comment (Edw1x) [inline src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.scss:235]: Combine this into one:
padding: 5px 0 0 10px

Comment (s1auka) [reply src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.scss]: done

File: src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.scss (hunk 2 above)
Comment (s1auka) [reply src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.scss]: done

**Review by Edw1x — APPROVED** [review]

File: src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+<div class="btn-group">
+  <div class="select-group">
+    <img [src]="ubsAdminTableIcons.arrowDown" alt="arrow down" />
+    <select class="display-counter" (change)="selectPageSize($event.target.value)">
+      <option [value]="size" *ngFor="let size of pageSizeOptions">{{ size }}</option>
+    </select>
+  </div>
+  <button type="button" class="btn btn-default" class="view-table-btn">
+    <mat-icon>create</mat-icon>
+    Додати сертифікат
```
Comment (rostikdoroshenko) [inline src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.html:14]: Can we move the text to .json files and use the translation here?

File: src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  </button>
+  <button type="button" class="btn btn-default" class="export-excel-btn">Експортувати в Excel</button>
```
Comment (rostikdoroshenko) [inline src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.html:16]: Can we move the text to .json files and use the translation here?

**Review by NewArtMask — APPROVED** [review]

**Review by Bishko86 — APPROVED** [review]

**Review by nazarhlyvka — APPROVED** [review]

File: src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.html (hunk 4 above)
Comment (s1auka) [reply src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.html]: done

File: src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.html (hunk 5 above)
Comment (s1auka) [reply src/app/ubs-admin/components/ubs-admin-certificate/ubs-admin-certificate.component.html]: done

**Review by rostikdoroshenko — APPROVED** [review]

**Pull request merged**